src/config/Chess.json

```diff
@@ -1,4 +1,5 @@
 {
+    "indexRenderer": "chess",
```

Comment:
I know it's total code bloat, but I would copy the whole file over and take this line out of Chess.json and put it in the new one. I've been trying to keep this as a very bare-bones version of MVP functionality - hence the number of variants of Chess.json in the project.

Reply:
This is a thing I care about.